src/memefactory/shared/utils.clj

```diff
@@ -7,3 +10,23 @@
     (binding [*out* *err*]
       (println "Building with environment:" env))
     env))
+
+(defmacro safe-go [& body]
```

Comment:
Shouldn't both macros be in https://github.com/district0x/async-helpers ?

Reply:
+1

Reply:
Done